src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  return {
+    add(a, b) {
+      return a + b;
```

Comment:
[CODE KNOWLEDGE] - When creating methods in an object, you don't need to use the function keyword, use the method shorthand instead. Also, the methods should be using the result property of the calculator object and directly modify it using the assignment operator with arithmetic operation (+=, -=, *=, /=) as per the checklist item 2. However, the add method should not take two parameters, it should only take one parameter and add it to the result property.